setup.py

```diff
@@ -32,16 +32,24 @@
 
 include_dirs = [numpy.get_include()]
 library_dirs = []
-libraries = ['gd']
+libraries = ['libgd']
 compile_args = []
 link_args = []
 
 if sys.platform.startswith('win'):
     # need the library and include for Windows Anaconda... <PREFIX>/Library
     include_dirs.append(os.path.join(sys.prefix, r'Library\include'))
+    #incpath = r'..\vcpkg\installed\x64-windows\include'
+    incpath = r'D:\WebGnome\vcpkg\installed\x64-windows\include'
+    include_dirs.append(os.path.join(sys.prefix, incpath))
+    print(" >>**<< INCLUDE PATH: ", os.path.join(sys.prefix, incpath)) 
+     
     # dlls go in bin, rather than lib (??)
     library_dirs.append(os.path.join(sys.prefix, r'Library\lib'))
-
+    libpath = r'D:\WebGnome\vcpkg\installed\x64-windows\lib'
```

Comment:
and one more ....

Reply:
I know, yet another hack. I'll need to find a standard way to add the path.  As it was, it seems that the development was done on Mac?